index/index.rst

```diff
@@ -30,7 +30,7 @@ This documents the *latest* ownCloud release, suitable for production use.
 ownCloud Server 6.0
 -------------------
 
-This documents the *previous* ownCloud release, suitable for production use.
+This documents the *older* ownCloud release, suitable for production use.
```

Comment:
maybe the real question is here what "deprecated" means. :-)
it is outdated and new users should install 8. Existing users should update to 8. But 6 is still supported with security fixes.